quarkus.jdt.ext/com.redhat.microprofile.jdt.quarkus/src/main/java/com/redhat/microprofile/jdt/internal/quarkus/scheduler/SchedulerUtils.java

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
+* Copyright (c) 2021 Red Hat Inc. and others.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Eclipse Public License v. 2.0 which is available at
+* http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+* which is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.microprofile.jdt.internal.quarkus.scheduler;
+
+import java.time.Duration;
+import java.time.format.DateTimeParseException;
+import java.util.regex.Pattern;
+
+/**
+ * Utility class for quarkus scheduler
+ *
+ */
+public class SchedulerUtils {
+
+	// Day of week and day of month cannot coexist
+	private static String[] cronPatterns = { "[*?0-59]", "[*?0-59]", "[*?0-23]", "[*?1-31]", "[*?1-12]",
```

Comment:
What about something like https://quarkus.io/guides/spring-scheduled#creating-a-scheduled-job ?

`@Scheduled(cron="*/5 * * * * ?")`

Reply:
Ah gotcha, what would this mean, every 5 seconds?

Reply:
If you click through the fields, you can see some of the supported values below.